src/main/java/evelyn/command/Ui.java

```diff
@@ -0,0 +1,102 @@
+package evelyn.command;
+
+import evelyn.exception.InvalidInputException;
+import evelyn.exception.NoInputException;
+
+import java.time.DateTimeException;
+import java.util.Objects;
+import java.util.Scanner;
+
+/**
+ * Houses all the logic for Evelyn's UI
+ */
+
+public class Ui {
+    public static String horizontalLine = "-----------------------------------------";
+    private boolean isChatting;
+    private String text = null;
+    private Parser parser;
+    private Scanner scanner;
+
+    /**
+     * Constructor of a Ui object.
+     * @param parser Parser used for the chatbot.
+     */
+    public Ui(Parser parser) {
+        this.isChatting = false;
+        this.scanner = new Scanner(System.in);
+        this.parser = parser;
+    }
+
+    /**
+     * Begins running the Ui.
+     */
+    public void start() {
+        this.isChatting = true;
+        System.out.println(horizontalLine);
+        System.out.println("Hi! I am evelyn.Evelyn");
```

Comment:
might be easier to maintain your code by compiling all the strings into a StringBuilder obj, then printing that instead